correlation/src/test/java/com/cerner/beadledom/correlation/CorrelationContextTest.java

```diff
@@ -0,0 +1,125 @@
+package com.cerner.beadledom.correlation;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class CorrelationContextTest {
+
+  CorrelationContext correlationContext = CorrelationContext.create();
+
+  @Nested
+  @DisplayName("#getId")
+  class Get {
+    @AfterEach
```

Comment:
Would it work if the `clearId` stuff was in the root class? Would it still run after each test of the nested classes?

Reply:
Yes, that works. I like it. 0770a372